### ISSUE TITLE
Expo client keeps crashing

### ISSUE THREAD

**Issue by ethanlal04** [expo/expo#10818]
Hello,

Please don't misunderstand me but it is really annoying I can't even test a project on my phone.

Each time I try running my project using the promo-code method it shows downloading some% and then 'network response timed out,  unable to connect to my IP address'. Then Samsung gives me this error 'expo client keeps crashing'.

I tried sending an email, connecting my phone to the same modem as my laptop, setting a certain 'environment variable' (which I am not sure if I did right), even reinstalling expo client. 

Whatever I do it shows my app is not running at the specific IP address. I don't know if the problem is in my project or in expo, but please try to fix it or at least officially announce how to fix the error, because a lot of people are getting the same throughout the web.

Please consider this,
Thanks, Ethan Lal.

**Comment by AdamJNavarro** [comment]
Hey @ethanlal04, can you provide some more information about the crash such as is this occurring on iOS or only Android? Are you running on a physical device or an emulator? Does this occur with a brand new project you've created or a specific project?

Cheers,
Adam

**Comment by ethanlal04** [comment]
I'm getting this error while running on an android physical device. I am using a windows machine with node v12.19.0 and Expo CLI version 3.28.2. Expo CLI used to work for the first time I used it, but then errors started coming and I was not able to run my project.

Then I moved to use an android emulator but I was getting another error 'command adb not found' (or something like that). Also, the emulator worked the first time. Then after the update to version 4 I started getting this error.

Thanks for replying, Ethan Lal
